graphql/database/users/getUserSearchEngine.js

```diff
@@ -33,7 +33,8 @@ const getUserSearchEngine = async (userContext, user) => {
     try {
       return getSearchEngine(JSON.parse(searchWidget.config).engine)
     } catch (e) {
-      // Don't care if SearchEngine does not exist if JSON is malformed.
+      // Don't care if SearchEngine does not exist. This will happen if
+      // the user has not explicitly set any search engine.
```

Comment:
or if search engine doesn't exist anymore (e.g. there's some disconnect between what we allowed in the Widget JSON and our searchEngineData file).